### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

Add `CODEOWNERS`. This list is open for comments, suggestions, additions, removals.

Resolves https://github.com/ESCOMP/CCPPStandardNames/issues/58

### CODE REVIEW

**Comment by climbfuji** [comment]
@dustinswales FYI

**Comment by climbfuji** [comment]
@svahl991 If you could help us with identifying a codeowner for the UK MetOffice please?

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+
+*       @cacraigucar    # NCAR/???
```
Comment (nusbaume) [inline CODEOWNERS:6]: Cheryl is NCAR/CGD

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+*       @climbfuji      # NRL
+*       @dustinswales   # NOAA/GSL
+*       @gold2718       # ???
+*       @grantfirl      # NOAA/GSL
+*       @mattldawson    # NCAR/???
```
Comment (nusbaume) [inline CODEOWNERS:11]: Matt is NCAR/ACOM

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+*       @mkavulich      # DTC
+*       @mwaxmonsky     # NCAR/CGD
+*       @nusbaume       # NCAR/CGD
+*       @peverwhee      # NCAR/CGD
+*       @svahl991       # JCSDA
+
+# Add codeowners for these organizations or remove comments before merging PR:
+# UKMO     ?
+# NOAA/EMC ?
+# NCAR/MMM ?
```
Comment (nusbaume) [inline CODEOWNERS:21]: I can try and find out who in MMM should be a code owner, but I don't think that should hold up this PR (as we can always add them later if need be).

**Review by nusbaume — APPROVED** [review]
Looks good to me!  I just posted a few clarifications to the comments.   Also I sent another invite to @dustinswales to try and give him write access to this repo, which I believe is required in order to be a code owner.

File: CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+
+*       @cacraigucar    # NCAR/???
+*       @climbfuji      # NRL
+*       @dustinswales   # NOAA/GSL
+*       @gold2718       # ???
```
Comment (gold2718) [inline CODEOWNERS:9]: `Meteorologisk institutt, Norway`
or maybe
`Norway MET`

**Review by gold2718 — APPROVED** [review]
I made a couple of suggestions for filling in my institution but am also approving so the PR can move forward.

File: CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+# These owners will be the default owners for everything in the repo.
+
+*       @cacraigucar    # NCAR/CGD
+*       @climbfuji      # NRL
+*       @dustinswales   # NOAA/GSL
+*       @gold2718       # Norway MET
+*       @grantfirl      # NOAA/GSL
+*       @mattldawson    # NCAR/ACOM
+*       @mkavulich      # DTC
```
Comment (mkavulich) [inline CODEOWNERS:12]:
```suggestion
*       @mkavulich      # NCAR/DTC
```

**Review by mkavulich — APPROVED** [review]
Thanks for making that change 👍

**Comment by svahl991** [comment]
> @svahl991 If you could help us with identifying a codeowner for the UK MetOffice please?

Please add both @MayeulDestouches and @ss421 for the Met Ofiice. Thanks.

File: CODEOWNERS [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+*       @grantfirl      # NOAA/GSL
+*       @mattldawson    # NCAR/ACOM
+*       @mkavulich      # NCAR/DTC
+*       @mwaxmonsky     # NCAR/CGD
+*       @nusbaume       # NCAR/CGD
+*       @peverwhee      # NCAR/CGD
+*       @svahl991       # JCSDA
+
+# Add codeowners for these organizations or remove comments before merging PR:
+# UKMO     ?
```
Comment (svahl991) [inline CODEOWNERS:19]: As mentioned elsewhere, @MayeulDestouches and @ss421 for UKMO.

**Review by cacraigucar — APPROVED** [review]

File: CODEOWNERS (hunk 6 above)
Comment (climbfuji) [reply CODEOWNERS:19]: Done!

**Review by svahl991 — APPROVED** [review]

**Review by mwaxmonsky — APPROVED** [review]

**Comment by climbfuji** [comment]
As discussed yesterday, I commented out ss421 as code owner because the invite to collaborate on the repo hasn't been accepted yet. We can add ss421 back in once that's done.

**Comment by climbfuji** [comment]
> > As discussed yesterday, I commented out ss421 as code owner because the invite to collaborate on the repo hasn't been accepted yet. We can add ss421 back in once that's done.
> 
> I have now accepted - apologies, I didn't spot the request.

Thanks! Added you back in. Will merge now.

**Pull request merged**